app/src/ui/branches/branches-container.tsx

```diff
@@ -261,4 +267,46 @@ export class BranchesContainer extends React.Component<
   ) => {
     this.setState({ selectedPullRequest })
   }
+
+  private getBranchWithName(branchName: string): Branch | undefined {
+    return this.props.allBranches.find(branch => branch.name === branchName)
+  }
+
+  private onRenameBranch = (branchName: string) => {
+    const branch: Branch | undefined = this.getBranchWithName(branchName)
```

Comment:
I think you can drop the variable type here now, right? 😌 
```suggestion
    const branch = this.getBranchWithName(branchName)
```